tutorons/common/api.py

```diff
@@ -0,0 +1,60 @@
+from tastypie.authorization import Authorization
+from tastypie.resources import ModelResource, ALL, ALL_WITH_RELATIONS
+
+from tutorons.common.models import Block, ServerQuery, ClientQuery, Region, ViewedRegion
+from tastypie import fields
+
+
+class BlockResource(ModelResource):
```

Comment:
Do we actually need to expose blocks, server queries, or regions through the API?  From looking at the `tutorons-library` code, I think all we need to expose are client queries and view events.